### PR TITLE
chore: Prep for release 3.10.1

### PR DESCRIPTION
**Pull request by zashraf1985**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 # Optimizely Ruby SDK Changelog
 
+## 3.10.1
+February 2, 2022
+
+### Enhancements:
+- Generate OptimizelyConfig object on API Call instead of SDK initialization to make the initialization faster (([#296](https://github.com/optimizely/ruby-sdk/pull/296)).
```
Comment (jaeopt) [inline CHANGELOG.md:7]: An error in the PR link reference format.

**Review by jaeopt — CHANGES_REQUESTED** [review]
nit

File: CHANGELOG.md (hunk 1 above)
Comment (zashraf1985) [reply CHANGELOG.md]: Good catch! just fixed it.

**Review by jaeopt — APPROVED** [review]
LGTM

**Pull request merged**